flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/functions/table/lookup/fullcache/LookupFullCache.java

```diff
@@ -72,6 +72,13 @@ public synchronized void open(Configuration parameters) throws Exception {
                                 } else {
                                     reloadFailCause.addSuppressed(th);
                                 }
+                                try {
```

Comment:
I used #close to be sure that reload won't happen again after exception. It was possible in case when there are no input records in interval between previous and next reloads. But actually it's not necessary to call #close, I changed the logic, so now flag `isStopped` in `CacheLoader` is being set after encountering an exception .